package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "judgemetest",
+  "version": "1.0.0",
+  "description": "Judge.Me Test Automation",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "judgeme": "npx cypress run",
+    "judgeme-headed": "npx cypress run --headed"
```

Comment:
Hmmm, I created custom scripts to provide flexibility for specific Cypress runs. I didn’t use the default test script to avoid potential conflicts with other tools or workflows that might expect a generic test command. Even though Cypress is installed in the workflow, using npx in the scripts ensures the locally installed version is used, maintaining consistency and avoiding dependency issues.